streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/SubscriptionInfoTest.java

```diff
@@ -25,19 +26,37 @@
 import java.util.Set;
 import java.util.UUID;
 
+import static org.apache.kafka.common.utils.Utils.mkEntry;
+import static org.apache.kafka.common.utils.Utils.mkMap;
 import static org.apache.kafka.streams.processor.internals.assignment.StreamsAssignmentProtocolVersions.LATEST_SUPPORTED_VERSION;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNull;
 
 public class SubscriptionInfoTest {
     private final UUID processId = UUID.randomUUID();
-    private final Set<TaskId> activeTasks = new HashSet<>(Arrays.asList(
+    private static final Set<TaskId> ACTIVE_TASKS = new HashSet<>(Arrays.asList(
         new TaskId(0, 0),
         new TaskId(0, 1),
         new TaskId(1, 0)));
-    private final Set<TaskId> standbyTasks = new HashSet<>(Arrays.asList(
+    private static final Set<TaskId> STANDBY_TASKS = new HashSet<>(Arrays.asList(
         new TaskId(1, 1),
         new TaskId(2, 0)));
+    private static final Map<TaskId, Integer> TASK_OFFSET_SUMS = mkMap(
+        mkEntry(new TaskId(0, 0), -1),
+        mkEntry(new TaskId(0, 1), -1),
+        mkEntry(new TaskId(1, 0), -1),
+        mkEntry(new TaskId(1, 1), 0),
+        mkEntry(new TaskId(2, 0), 10)
+    );
+    static {
+        TASK_OFFSET_SUMS.put(new TaskId(0, 0), -1);
+        TASK_OFFSET_SUMS.put(new TaskId(0, 1), -1);
+        TASK_OFFSET_SUMS.put(new TaskId(1, 0), -1);
+        TASK_OFFSET_SUMS.put(new TaskId(1, 1), 0);
+        TASK_OFFSET_SUMS.put(new TaskId(2, 0), 10);
+    }
```

Comment:
What's up with the pre-loaded map followed by a static block adding the same contents to it? (Looks like maybe a refactoring artifact).

Reply:
Technically it was a "commit straight from github suggestion" artifact 🙂